platform/commonUI/edit/src/actions/CancelAction.js

```diff
@@ -82,7 +66,7 @@ define(
         CancelAction.appliesTo = function (context) {
             var domainObject = (context || {}).domainObject;
             return domainObject !== undefined &&
-                domainObject.hasCapability("editor");
+                domainObject.getCapability("status").get("editing");
```

Comment:
Yes; the distinction I'm seeing:

1. `isEditingContextRoot`: Identifies the specific domain object used to initiate editing.
2. `inEditingContext`: Identifies any domain object that is part of the editing context (either because it is the specific domain object being edited, or because it is a sub-object being edited, e.g. in a timeline)

My reading was that the `editing` status was being used to distinguish the first case, and `inEditingContext` was being used to identify the more general second case. If they're actually identical, I would stick with the existing `inEditingContext` because it's simpler/clearer, but I would also use that check in lieu of checking status.